specs/MultiProfile.md

```diff
@@ -22,6 +22,12 @@ multiple WebView2s running with separate profiles under a single user data direc
 browser instance at runtime), which means separate cookies, user preference settings, and various
 data storage etc., to help you build a more wonderful experience for your application.
 
+In order to manage cookies through profile, we're adding a get_CookieManager interface into profile.
+Users can use this interface to get the cooke manager, which is shared by all webview2s associated with
```

Comment:
"cooke" -> "cookie", fix capitalization of WebView2.

Reply:
Sorry for the misspellings, thanks!